src/hooks/useUpdateSubscription.tsx

```diff
@@ -21,7 +21,7 @@ export const useUpdateSubscription = ({
   pricePlanName,
   baseApiUrl,
   options = {},
-}: Props): UseAsyncOnDemandReturnType<ActiveSubscription> => {
+}: Props): UseAsyncOnDemandReturnType<ActivePricePlan> => {
```

Comment:
Shouldn't this return ActiveSubscription?

Reply:
Decided this offline: 
We want to update `updateSubscription` endpoint to return the same object as for `getActiveSubscription`. 
The returned type will be renamed after it.